sale_coupon_criteria_multi_product/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2021 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Coupons multi product criteria",
+    "summary": "Allows set diferent criteria rules for specific products",
+    "version": "13.0.1.0.0",
+    "development_status": "Beta",
```

Comment:
Having tests, it can be:
```suggestion
    "development_status": "Production/Stable",
```